driver-core/src/main/com/mongodb/internal/binding/SingleConnectionReadBinding.java

```diff
@@ -1,116 +0,0 @@
-/*
- * Copyright 2008-present MongoDB, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.mongodb.internal.binding;
-
-import com.mongodb.ReadPreference;
-import com.mongodb.connection.ServerDescription;
-import com.mongodb.internal.connection.Connection;
-import com.mongodb.internal.connection.NoOpSessionContext;
-import com.mongodb.internal.session.SessionContext;
-
-import static com.mongodb.assertions.Assertions.notNull;
-
-/**
- * A read binding that is bound to a single connection.
- *
- * @since 3.2
- */
-public class SingleConnectionReadBinding extends AbstractReferenceCounted implements ReadBinding {
```

Comment:
Rather than add ServerApi to an unused class, I just removed it.

Reply:
Ack :+1: